client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-identify/index.tsx

```diff
@@ -23,8 +25,20 @@ interface Props {
 
 export const Analyzer: FC< Props > = ( { onComplete, onSkip, hideImporterListLink = false } ) => {
 	const translate = useTranslate();
+	const isEnglishLocale = useIsEnglishLocale();
 	const [ siteURL, setSiteURL ] = useState< string >( '' );
 
+	// TODO: Remove extra steps for non-English locales once we have translations.
```

Comment:
Should we create an issue for that or just leave as a TODO and remove it "eventually"?

Reply:
Donna and I chatted a bit about it and we also discussed in our team meeting. Probably creating issue will create too much noise and be a bit overwhelming. As it's not very significant, a TODO seemed a milder way of indication